packages/frontend/src/hooks/usePositions.ts

```diff
@@ -514,6 +514,7 @@ export const usePnL = () => {
   const currentShortDeposits = useMemo(() => {
     if (positionType === PositionType.LONG) return []
     let totalShortSqth = new BigNumber(0)
+    //@ts-expect-error
```

Comment:
hmm whats this?

Reply:
Oh I forgot to remove it. It's a typescript error suppressor. I'll remove it